modules/custom/az_dam/config/quickstart/field.field.media.ib_dam_embed.field_media_ib_dam_embed.yml

```diff
@@ -0,0 +1,25 @@
+uuid: 315a190d-a53f-4eb9-9e0f-9e3a95b8f13b
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.media.field_media_ib_dam_embed
+    - media.type.ib_dam_embed
+  module:
+    - link
+_core:
+  default_config_hash: p1KexYTRAx9iPcnr_0I3rToTWFuUinxf2M7EZeNAJig
```

Comment:
```suggestion
```

Reply:
Resolved locally. Will commit for review when all changes have been made.